mkdocs.yml

```diff
@@ -123,7 +123,8 @@ nav:
             - Constantin: tools/constantin.md
             - OracleBP: tools/oraclebp.md
             - NEMU: tools/nemu.md
-            - 编译 Linux Kernel: tools/linux-kernel-for-xs.md
+            - 编译使用 RISCV-pk 的 Linux Kernel: tools/linux-kernel-for-xs.md
```

Comment:
Should we deprecate the use of riscv-pk in XiangShan doc? I think we don’t need it if we have OpenSBI to use.

Reply:
i think we can keep it for now and consider deprecate it again once we have fully migrate to OpenSBI.

for example, NEMU master branch only support simpoint and checkpoint using riscv-pk

Reply:
OK.